Gems/LmbrCentral/Code/include/LmbrCentral/Shape/ShapeComponentBus.h

```diff
@@ -174,6 +191,19 @@ namespace LmbrCentral
             return false;
         }
 
+        /// Get the translation offset for the shape relative to its entity.
+        virtual AZ::Vector3 GetTranslationOffset() const
+        {
+            AZ_Warning("ShapeComponentRequests", false, "GetTranslationOffset not implemented");
+            return AZ::Vector3::CreateZero();
+        }
+
+        /// Set the translation offset for the shape relative to its entity.
+        virtual void SetTranslationOffset([[maybe_unused]] const AZ::Vector3& translationOffset)
+        {
+            AZ_Warning("ShapeComponentRequests", false, "SetTranslationOffset not implemented");
+        }
```

Comment:
`GetTranslationOffset` and `SetTranslationOffset` shouldn't through a warning if translation is disabled.